PC/launcher.c

```diff
@@ -1412,22 +1412,29 @@ show_help_text(wchar_t ** argv)
 Python Launcher for Windows Version %ls\n\n", version_text);
     fwprintf(stdout, L"\
 usage:\n\
-%ls [launcher-args] [python-args] script [script-args]\n\n", argv[0]);
+%ls [launcher-args] [python-args] [script [script-args]]\n\n", argv[0]);
     fputws(L"\
 Launcher arguments:\n\n\
--2     : Launch the latest Python 2.x version\n\
--3     : Launch the latest Python 3.x version\n\
+-2     : Launch the latest* Python 2.x version\n\
+-3     : Launch the latest* Python 3.x version\n\
 -X.Y   : Launch the specified Python version\n", stdout);
     if (canDo64bit) {
         fputws(L"\
-     The above all default to 64 bit if a matching 64 bit python is present.\n\
+     The above all default to 64 bit if a matching 64 bit python is present*.\n\
 -X.Y-32: Launch the specified 32bit Python version\n\
--X-32  : Launch the latest 32bit Python X version\n\
+-X-32  : Launch the latest* 32bit Python X version\n\
 -X.Y-64: Launch the specified 64bit Python version\n\
--X-64  : Launch the latest 64bit Python X version", stdout);
+-X-64  : Launch the latest* 64bit Python X version", stdout);
     }
     fputws(L"\n-0  --list       : List the available pythons", stdout);
     fputws(L"\n-0p --list-paths : List with paths", stdout);
+    fputws(L"\n\n If no script is specified the specified interpreter is opened.", stdout);
+    fputws(L"\n*If an exact version is not given latest can be overriden by:", stdout);
+    fputws(L"\n An active virtual environment", stdout);
+    fputws(L"\n A shebang line in the script (if present)", stdout);
+    fputws(L"\n Matching PY_PYTHON[n] Enviroment variable(s)", stdout);
```

Comment:
Do you think it's clear that `PY_PYTHON` sets the default and `PY_PYTHON2` and `PY_PYTHON3` are for the -2 and -3 options?

Reply:
Reworded to make clearer